eta/core/storage.py

```diff
@@ -613,6 +614,46 @@ def delete(self, file_or_folder_id):
         self._service.files().delete(
             fileId=file_or_folder_id, supportsTeamDrives=True).execute()
 
+    def delete_all_files_in_folder(self, folder_id, skip_failures=False):
+        '''Deletes all the files in the Google Drive folder and retains the
+        original folder.
+
+        Args:
+            folder_id: the ID of the Google Drive folder from which the files
+                are to be deleted
+            skip_failures: whether to gracefully skip delete errors. By
+                default, this is False
+        '''
+        files = self.list_files_in_folder(folder_id)
+        for f in files:
+            filename = f["name"]
+            try:
+                with etau.Timer() as t:
+                    self.delete(f["id"])
+                    logger.info(
+                        "File '%s' deleted from '%s' (%s)", filename,
+                        folder_id, t.elapsed_time_str)
+            except Exception as e:
+                if not skip_failures:
+                    raise GoogleDriveStorageClientError(e)
+                    logger.info(
+                        "Failed to delete file '%s' from '%s' (%s)", filename,
+                        folder_id, t.elapsed_time_str)
+
+        if not len(self.list_files_in_folder(folder_id)):
+            logger.info("All files deleted in %s", folder_id)
+
+    def count_files_in_folder(self, folder_id):
```

Comment:
Do you need to have this method now?  It doesn't look like you're actually using it.

Reply:
it would be good to have a count method, but not mandatory though. It can removed, if not necessary.